pybryt/annotations/value.py

```diff
@@ -284,6 +282,15 @@ def check_values_equal(value, other_value, atol = None, rtol = None, equivalence
                         if len(value) != len(other_value):
                             return False
                         res = np.array(np.isclose(v, o, atol=atol, rtol=rtol) for v, o in zip(sorted(value), sorted(other_value))).all()
+                    elif isinstance(value, dict):
+                        if not isinstance(other_value, dict):
+                            return False
+                        if all(isinstance(i, numbers.Real) for i in value.values()):
+                            res = np.array(np.isclose(k1, k2) and np.isclose(v1, v2) \
+                                for (k1, v1), (k2, v2) in zip(sorted(value.items(), key=lambda t: t[0]), \
```

Comment:
very clever :-) but someone coming into this code in the future will have to spent a while figuring out what this does. Please either refactor explode this, or add some comments to explain what you're doing